### PR TITLE
font

### PR DESCRIPTION
**Pull request by Aki-7**
## Context

Introduce system level font configuration.

## Summary

Hold font faces at global level.

## How to check behavior

No behavioral change is made

### CODE REVIEW

File: common/font.c [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+#include "zen-common/log.h"
+#include "zen-common/util.h"
+
+static struct zn_font *font_singleton = NULL;
+
+static const char *font_list[ZN_FONT_TYPE_COUNT] = {
+    [ZN_FONT_REGULAR] = "Ubuntu:style=Regular",
+    [ZN_FONT_BOLD] = "Ubuntu:weight=200",
+};
```
Comment (Aki-7) [inline common/font.c:15]: Font is specified by "<family-name>:<property>=<value>".

See https://www.freedesktop.org/software/fontconfig/fontconfig-user.html for properties and values details

Comment (liwii) [reply common/font.c:15]: Isn't it possible to write “Ubuntu:bold” ? I took a look at the doc a little bit, and it seems like “bold” is equivalent to “weight=200”. See “Font Name” and “Configuration File Format” -> “<constant>”.

Comment (Aki-7) [reply common/font.c:15]: It can be. I'll check it later and fix it if possible. Thanks!

**Review by watasuke102 — APPROVED** [review]
Nice work!

File: common/font.c (hunk 1 above)
Comment (Aki-7) [reply common/font.c]: It works, fixed to `Ubuntu:bold`. Thanks!

**Pull request merged**